packages/fx-core/src/common/local/localCertificateManager.ts

```diff
@@ -90,22 +90,25 @@ export class LocalCertificateManager {
         }
       }
 
+      localCert.found = !!certThumbprint;
       if (!certThumbprint) {
         if (failIfNotFound) {
-          localCert.found = false;
           return localCert;
         }
         // generate cert and key
         certThumbprint = await this.generateCertificate(certFilePath, keyFilePath);
       }
 
       if (needTrust) {
-        if (certThumbprint && (await this.verifyCertificateInStore(certThumbprint))) {
```

Comment:
Add this back? There's corner case `certThumbprint` undefined here due to generate error.